pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -585,7 +585,7 @@ def test_ms_single_chan(mir_uv, future_shapes, tmp_path):
         ms_uv.read(testfile)
     assert str(cm.value).startswith("No valid data available in the MS file.")
 
-    ms_uv.read(testfile, ignore_single_chan=False, read_weights=False)
+    ms_uv.read_ms(testfile, ignore_single_chan=False)
```

Comment:
I think this might be a rebase mistake, unless you really want to make this change. I changed it as few PRs ago from `read_ms` to `read` with the `read_weights` keyword to test that everything was properly piped through to the generic read method.

Reply:
Yeah, this was a mistake in rebase -- just meant to nuke the `read_weights` keyword. Fixed in 2d0e86b6.